lib/galaxy/webapps/config_manage.py

```diff
@@ -51,14 +51,84 @@
 
 UWSGI_OPTIONS = OrderedDict([
     ('http', {
-        'desc': """The address and port on which to listen.  By default, only listen to localhost ($app_name will not be accessible over the network).  Use '0.0.0.0' to listen on all available network interfaces.""",
+        'desc': """The address and port on which to listen.  By default, only listen to localhost ($app_name will not be accessible over the network).  Use ':$default_port' to listen on all available network interfaces.""",
         'default': '127.0.0.1:$default_port',
         'type': 'str',
     }),
+    ('processes', {
+        'desc': """Number of web server (worker) processes to fork after the application has loaded.""",
+        'default': '1',
+        'type': 'int',
+    }),
     ('threads', {
-        'default': 8,
+        'desc': """Number of threads for each web server process.""",
+        'default': 4,
+        'type': 'int',
+    }),
+    ('offload-threads', {
+        'desc': """Number of threads for serving static content and handling internal routing requests.""",
+        'default': '2',
         'type': 'int',
     }),
+    ('static-map.1', {
+        'key': 'static-map',
+        'desc': """Mapping to serve style content.""",
+        'default': '/static/style=static/style/blue',
+        'type': 'str',
+    }),
+    ('static-map.2', {
+        'key': 'static-map',
+        'desc': """Mapping to serve the remainder of the static content.""",
+        'default': '/static=static',
+        'type': 'str',
+    }),
+    ('master', {
+        'desc': """Enable the master process manager. Disabled by default for maximum compatibility with CTRL+C, but should be enabled for use with --daemon and/or production deployments.""",
+        'default': 'false',
+        'type': 'str',
```

Comment:
@jmchilton if I set this to `bool` then the values in the sample are pythonic (e.g. `True` and `False`).